static/js/controllers/configuration-upgrade.js

```diff
@@ -0,0 +1,110 @@
+var PouchDB = require('pouchdb-browser');
```

Comment:
OK I'm just importing angular's pouchdb int. for now. I don't see any benefit in refactoring the DB service, it's very centred on self-identifying DBs relevant to hostname etc, and doesn't provide any benefits if you're not doing that.